Storage/Storage/CoreData/CoreDataManager.swift

```diff
@@ -7,48 +7,69 @@ import WooFoundation
 ///
 public final class CoreDataManager: StorageManagerType {
 
-    /// Storage Identifier.
-    ///
-    public let name: String
-
     private let crashLogger: CrashLogger
 
-    private let modelsInventory: ManagedObjectModelsInventory
-
     /// A serial queue used to ensure there is only one writing operation at a time.
     private let writerQueue: OperationQueue
 
+    let storeURL: URL
+
+    let storeDescription: NSPersistentStoreDescription
+
```

Comment:
nit;

```suggestion
    private let storeURL: URL

    private let storeDescription: NSPersistentStoreDescription

```

Reply:
We're currently having [a unit test](https://github.com/woocommerce/woocommerce-ios/blob/c3edcbaf2d6fcbd0a844af7240d8462183e040fa/Storage/StorageTests/CoreData/CoreDataManagerTests.swift#L15-L19) that checks for these - that's why I keep them internal. Since this doesn't cause any issue in terms of safety, I'd like to keep it as-is. WDYT?

Reply:
Oh, I missed that. All good 👍